audit_trail.go

```diff
@@ -0,0 +1,104 @@
+package tfe
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"time"
+)
+
+// Compile-time proof of interface implementation.
+var _ AuditTrails = (*auditTrails)(nil)
+
+// AuditTrails describes all the audit trail related methods that the
+// Terraform Cloud API supports.
+//
+// TFE API docs: https://www.terraform.io/docs/cloud/api/audit-trails.html
+type AuditTrails interface {
+	// List audit trails visible to the current user.
+	List(ctx context.Context, options AuditTrailListOptions) (*AuditTrailList, error)
```

Comment:
```suggestion
	// List an organization's audit events. Requires an organization token.
	ListAuditEvents(ctx context.Context, options AuditEventListOptions) (*AuditEventList, error)
```

Reply:
Sure thing, I understand. It is best to get the naming right before moving on!